app/Policies/FloorPolicy.php

```diff
@@ -24,56 +24,52 @@ public function viewAny(MoonshineUser $user): bool
 
     public function view(MoonshineUser $user, Floor $item): bool
     {
-        if ($user->moonshineUserRole->id === Constants::ROLES['Admin']) {
-            return true;
+        if ($user->moonshineUserRole->id === Constants::ROLES['Developer']) {
+            return $item->created_by === $user->id;
```

Comment:
Floor does not have `created_by` property (column in db). Change to `$item->hotel->created_by`.